src/main/webapp/app/entities/course/course.service.ts

```diff
@@ -320,32 +320,50 @@ export class CourseScoreCalculationService {
     constructor(private dateUtils: JhiDateUtils, private http: HttpClient) { }
 
     calculateTotalScores(courseExercises: Exercise[]): Map<string, number> {
-        let scores = new Map<string, number>();
-        let absoluteScore = 0;
-        let relativeScore = 0;
+        const scores = new Map<string, number>();
+        let absoluteScore = 0.0;
         let maxScore = 0;
-        for (let i = 0; i < courseExercises.length; i++) {
-            const exercise = courseExercises[i];
-            if (exercise.maxScore !== null) {
-                maxScore = maxScore + exercise.maxScore;
-                const participation: Participation = this.getParticipationForExercise(exercise);
-                if (participation !== undefined) {
-                    const result: Result = this.getResultForParticipation(participation, exercise.dueDate);
-                    absoluteScore = absoluteScore + result.score * this.SCORE_NORMALIZATION_VALUE * exercise.maxScore;
-                    relativeScore = relativeScore + result.score;
-                }
-            }
-        }
-        scores.set('absoluteScore', Math.round(absoluteScore * 100) / 100);
+        courseExercises.forEach( exercise => {
+          if (exercise.maxScore !== null) {
+              maxScore = maxScore + exercise.maxScore;
+              const participation: Participation = this.getParticipationForExercise(exercise);
+              if (participation !== undefined) {
+                  const result: Result = this.getResultForParticipation(participation, exercise.dueDate);
+                  absoluteScore = absoluteScore + result.score;
```

Comment:
Could you please revert the formula again? The absolute score now is wrongly calculated...